src/index.tsx

```diff
@@ -6,9 +7,13 @@ import MulticallUpdater from './state/multicall/updater'
 import TransactionUpdater from './state/transactions/updater'
 
 export function Updaters() {
+  const router = useRouter()
+  const includeListUpdater = ['/swap', '/limit-orders', '/add', '/find', '/remove'].some((item) => {
```

Comment:
when user go to `swap`, go to `farm` and then go back to `swap`, the whole thing in `ListsUpdater` will be executed again, right?

Reply:
You are right, I need to change the logic to not make request every time page with currency search visited

Reply:
Done 👍